SCF-Session-9-Assignment/src/question/Circle.java

```diff
@@ -0,0 +1,100 @@
+package question;
+
+import java.util.Date;
+
+/**
+ * This class implements the Shape interface.
+ * This class overrides the methods of Shape interface to perform desired Operations
+ * @author Jugal
+ * Dated 07/24/2019
+ */
+public class Circle implements Shape {
+	
+	double xOrigin, yOrigin, radiusOfCircle;
+	Date timeStamp;
+	int shapeId;

```

Comment:
For all type shape classes declare the shapeId in Shape interface.

Reply:
In interface, the variables declared become final.
And shapeId will be different for all the shapes.
That's why declared individually.